src/index.html

```diff
@@ -1,13 +1,333 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MYBIKE</title>
+
+    <link 
+    rel="icon" 
+    type="image/x-icon" 
+    href="images/logo.png"
+    >
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+    rel="preconnect" 
+    href="https://fonts.gstatic.com" 
+    crossorigin="use-credentials"
+    >
+    <link 
+    href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" 
+    rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+
+
+  <!-- #endregion -->
+
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo">
+          <img src="./images/logo.png" alt="MyBike logo">
+        </a>
+
+        <div class="top-bar__icons">  
+          <a class="icon icon--phone" href="#"></a>
+          <a class="icon icon--menu" href="#menu"></a>
+        </div>
+      </div>
+
+      <div class="header__bottom">
+        <h1 class="header-title">Take the Streets</h1>
+      </div>
+    </header>
+
+    <aside class="menu page__menu" id="menu">
+      <div class="top-bar menu__top">
+        <a href="#" class="top-bar__logo">
+          <img src="./images/logo-dark.png" alt="MyBike logo">
+        </a>
+
+
+        <div class="header__icons">  
+          <a class="icon icon--close" href="#"></a>
+        </div>
+      </div>
+
+      <div class="menu-content">
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link" href="#">HOME</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#about">ABOUT US</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#bikes">COMPARE BIKES</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#details">DETAILS</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#contacts">CONTACTS</a>
+            </li>
+          </ul>
+        </nav>
+
+        <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
+          +1 234 5555-55-55
+        </a>
+  
+        <a href="tel:+1 234 5555-55-55" class="menu__call-to-book">
```

Comment:
Bad value tel:+1 234 5555-55-55 for attribute href on element [a](https://html.spec.whatwg.org/multipage/#the-a-element): Illegal character in scheme data: space is not allowed.